react/Table/Virtualized/Dnd/TableRow.jsx

```diff
@@ -83,7 +84,7 @@ const TableRow = ({ item, context, ...props }) => {
     <TableRowClassic
       {...props}
       ref={node => dragRef(dropRef(node))}
-      selected={isSelected || dropCollect.isOver}
+      selected={isSelected || dropCollect.isOver || isNew}
```

Comment:
As we finally use `selected` prop from `Mui TableRow`, I don't think we have to change anything in cozy-ui, isn't it? Could we just have to deal with `isSelectedItem` main prop (maybe it conflicts with DnD... 🤔) in the app?

Reply:
I agree, that's why we decided to not use `selected` prop in the first place and to add a specific css or something for it, if I remember correctly.